### PR TITLE
Fix #14  msgs.Messages undefined

### LINKED ISSUE #14: Docker build fails with msgs.Messages undefined (type []*vici.Message has no field or method Messages)
Hi, 

Docker build fails with `msgs.Messages undefined`.

```
22.06 ./main.go:30:24: msgs.Messages undefined (type []*vici.Message has no field or method Messages)
------
Dockerfile:9
--------------------
   7 |     #	go mod download
   8 |     COPY app /app
   9 | >>> RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o main .
  10 |
  11 |     FROM alpine:latest
--------------------
ERROR: failed to solve: process "/bin/sh -c CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o main ." did not complete successfully: exit code: 2
```

### PR DESCRIPTION
**Pull request by oschwartz10612**
Just had to alter the function a bit to work with the vici lib changes. How does this look to you? 

Tested and working!

### CODE REVIEW

File: app/main.go [hunk 1]
```diff
@@ -27,22 +28,25 @@ func listSAs()([]LoadedIKE, error){
...
 				continue
 			}
 			ike.Name = k
 			retVar = append(retVar, ike)
-
 		}
 	}
+
+	// close the socket
+	s.Close()
```
Comment (LarsTi) [inline app/main.go]: This should not be used, as line 24 already registers the socket to be closed once the scope is left.

File: app/main.go [hunk 2]
```diff
@@ -27,22 +28,25 @@ func listSAs()([]LoadedIKE, error){
 	if err != nil {
 		return retVar, err
 	}
-	for _,m := range msgs.Messages() {
-		if e := m.Err(); e != nil{
+	for _, m := range msgs { // <- Directly iterate over msgs
```
Comment (LarsTi) [inline app/main.go:31]: Do you know, why msgs.Messages() does not bring the expected result?
I didn't see any changes in the api regarding this, but i may be blind.

**Review by LarsTi — COMMENTED** [review]
Hey there,

glad you found the tool useful.
Unfortunately i can check your changes not before next week, but i appreciate your work! :)

I will have a look at it

File: app/main.go (hunk 1 above)
Comment (oschwartz10612) [reply app/main.go]: Good point thanks. Ill update this to remove the close.

File: app/main.go (hunk 2 above)
Comment (oschwartz10612) [reply app/main.go:31]: I do not, but this appears to get the job done. Odd change I agree!

**Comment by oschwartz10612** [comment]
Thanks! I appreciate the project and that you provided a dashboard! 

I updated to remove the close. No rush on the review and merge!

**Comment by fengmikaelson** [comment]
is this commit merge?

**Comment by LarsTi** [comment]
sorry, i rly forgot about this thing.
I merged into main and will issue a new release, so there is a new docker image.

**Pull request merged**